cypress/e2e/tests/migration/dynamic-report/issues/allIssues/sort.test.ts

```diff
@@ -0,0 +1,56 @@
+import { Analysis } from "../../../../../models/migration/applicationinventory/analysis";
+import {
+    deleteByList,
+    getRandomAnalysisData,
+    getRandomApplicationData,
+    login,
+    validateSortBy,
+} from "../../../../../../utils/utils";
+import { SEC } from "../../../../../types/constants";
+import { Issues } from "../../../../../models/migration/dynamic-report/issues/issues";
+
+describe(["@tier2"], "Issues sort validations", function () {
+    let applicationsList: Array<Analysis> = [];
```

Comment:
**question:** Why do you create a list if you're working with only one application?

Reply:
Removed